cmd/nodeup/main.go

```diff
@@ -101,7 +101,6 @@ func main() {
 			i := bootstrap.Installation{
 				CacheDir: flagCacheDir,
 				Command:  command,
-				FSRoot:   flagRootFS,
```

Comment:
You should probably also remove the `rootfs` flag.

Reply:
Good point. Forgot about it.